indico/modules/rb_new/client/js/modules/blockings/reducers.js

```diff
@@ -46,13 +46,33 @@ export default combineReducers({
             blockingsActions.CREATE_BLOCKING_SUCCESS,
             blockingsActions.CREATE_BLOCKING_ERROR
         ),
+        blockingState: requestReducer(
+            blockingsActions.CHANGE_BLOCKING_STATE_REQUEST,
+            blockingsActions.CHANGE_BLOCKING_STATE_SUCCESS,
+            blockingsActions.CHANGE_BLOCKING_STATE_ERROR,
+        ),
+        delete: requestReducer(
+            blockingsActions.DELETE_BLOCKING_REQUEST,
+            blockingsActions.DELETE_BLOCKING_SUCCESS,
+            blockingsActions.DELETE_BLOCKING_ERROR,
+        ),
     }),
     blockings: (state = {}, action) => {
         switch (action.type) {
             case blockingsActions.BLOCKINGS_RECEIVED:
                 return action.data.reduce((obj, b) => ({...obj, [b.id]: camelizeKeys(b)}), {});
             case blockingsActions.BLOCKING_RECEIVED:
                 return {...state, [action.data.id]: camelizeKeys(action.data)};
+            case blockingsActions.CHANGE_BLOCKING_STATE_SUCCESS: {
+                const {blocking} = camelizeKeys(action.data);
+                return {...state, [blocking.id]: blocking};
+            }
+            case blockingsActions.DELETE_BLOCKING_SUCCESS: {
+                const {blockingId} = camelizeKeys(action.data);
+                const newState = {...state};
```

Comment:
An alternative way to do this would be:

    return _.omit(state, blockingId)

(not sure if it needs to be converted to a string first, and nothing wrong with the current way you are doing it)